cinema/views.py

```diff
@@ -183,8 +217,6 @@
     )
     serializer_class = OrderSerializer
     pagination_class = OrderPagination
-    authentication_classes = (TokenAuthentication,)
-    permission_classes = (IsAuthenticated,)
 
     def get_queryset(self):
         return Order.objects.filter(user=self.request.user)
```

Comment:
The `get_queryset` method in `OrderViewSet` filters orders by the current user, which is a good practice for ensuring that users can only access their own orders.